### PR TITLE
Update stNEAR token metadata: Update icon and add Aurora address

### PR DESCRIPTION
**Pull request by camchis**
# Update stNEAR metadata

NEP-141: https://explorer.near.org/accounts/meta-pool.near

### CODE REVIEW

File: tokens/stnear.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 288 288"><g id="Layer_1" data-name="Layer 1"><path d="M187.58,79.81l-30.1,44.69a3.2,3.2,0,0,0,4.75,4.2L191.86,103a1.2,1.2,0,0,1,2,.91v80.46a1.2,1.2,0,0,1-2.12.77L102.18,77.93A15.35,15.35,0,0,0,90.47,72.5H87.34A15.34,15.34,0,0,0,72,87.84V201.16A15.34,15.34,0,0,0,87.34,216.5h0a15.35,15.35,0,0,0,13.08-7.31l30.1-44.69a3.2,3.2,0,0,0-4.75-4.2L96.14,186a1.2,1.2,0,0,1-2-.91V104.61a1.2,1.2,0,0,1,2.12-.77l89.55,107.23a15.35,15.35,0,0,0,11.71,5.43h3.13A15.34,15.34,0,0,0,216,201.16V87.84A15.34,15.34,0,0,0,200.66,72.5h0A15.35,15.35,0,0,0,187.58,79.81Z"/></g></svg>
```
Comment (sept-en) [inline tokens/stnear.svg:1]: What's the reason to update the icon? And where can we verify that it's actually was officially updated?

File: tokens/stnear.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
     "ethereum_address": "",
-    "aurora_address": "",
+    "aurora_address": "0x07f9f7f963c5cd2bbffd30ccfb964be114332e30",
     "near_address":"meta-pool.near",
     "name": "Staked NEAR",
-    "symbol": "STNEAR",
+    "symbol": "stNEAR",
```
Comment (sept-en) [inline tokens/stnear.json:6]: this should match the `ft_metadata()` in NEAR smart contract which contains "STNEAR", not "stNEAR"

File: tokens/stnear.svg (hunk 1 above)
Comment (camchis) [reply tokens/stnear.svg:1]: The previous icon was incorrect, the new icon is the same as the normal NEAR token icon and wNEAR icon

File: tokens/stnear.json (hunk 2 above)
Comment (camchis) [reply tokens/stnear.json]: Thanks, updated

File: tokens/stnear.svg (hunk 1 above)
Comment (sept-en) [reply tokens/stnear.svg:1]: stNEAR should have its separate icon, moreover, it has that separate icon in `ft_metadata()` for NEAR token. So we shouldn't use another icon for Aurora tokens list. If you have any thoughts on this matter I recommend contacting Meta-Pool team first and discussing it with them.

**Comment by sept-en** [comment]
Closed because of the following comment: https://github.com/aurora-is-near/bridge-assets/pull/71/files#r777606916

**Comment by camchis** [comment]
Can we at least add the Aurora address? So that we can actually bridge stNEAR to Aurora?

**Pull request closed without merging**